sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/ShareFileAsyncClient.java

```diff
@@ -1248,7 +1261,11 @@ private void channelCleanUp(AsynchronousFileChannel channel) {
      * @param data The data which will upload to the storage file.
      * @param length Specifies the number of bytes being transmitted in the request body.
      * @return A response that only contains headers and response status code
+     *
+     * @deprecated Use {@link ShareFileAsyncClient#uploadRange(Flux, long)} instead. Or consider
+     * {@link ShareFileAsyncClient#upload(Flux, ParallelTransferOptions)} for a more capable upload method.
```

Comment:
can we explicitly call out "for an upload method that can handle uploading large amounts of data" or something like that?